packages/git/src/browser/history/git-history-widget.tsx

```diff
@@ -124,18 +124,15 @@ export class GitHistoryWidget extends GitNavigableListWidget<GitHistoryListNode>
     protected async addCommits(options?: Git.Options.Log): Promise<void> {
         let repository: Repository | undefined;
         repository = this.repositoryProvider.findRepositoryOrSelected(options);
-        let resolver: () => void;
+
         this.errorMessage = undefined;
         this.cancelIndicator.cancel();
         this.cancelIndicator = new CancellationTokenSource();
         const token = this.cancelIndicator.token;
+
         if (repository) {
-            const log = this.git.log(repository, options);
-            log.catch((reason: Error) => {
-                this.errorMessage = reason.message;
-                resolver();
-            });
-            log.then(async changes => {
+            try {
+                let changes = await this.git.log(repository, options);
```

Comment:
The previous state did not have the `await`. Are you OK with the change, @jbicker?

Reply:
I'll have a look now.

Reply:
Ok, works as before. I am ok with the change as it actually simplifies the code. 
Thanks you, @vince-fugnitto!

Reply:
👍

Reply:
thanks @jbicker !